src/app/clusters/groupcast-server/GroupcastServer.cpp

```diff
@@ -0,0 +1,105 @@
+#include "GroupcastServer.h"
+
+
+using chip::Protocols::InteractionModel::Status;
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace {
+static GroupcastCluster sInstance;
+} // namespace
+
+GroupcastCluster & GroupcastCluster::Instance()
+{
+    return sInstance;
```

Comment:
Can we make the app handle these instead?

And in zap case we can have CodegenIntegration.cpp provide it